docs/source/algorithms/FFTPreProcessing-v1.rst

```diff
@@ -0,0 +1,93 @@
+.. algorithm::
+
+.. summary::
+
+Allows the user to generate data that has padding (extra measurements of zero) and/or apply an apodization function.
+
+.. alias::
+
+.. properties::
+
+
+Description
+-----------
+
+This algorithm can prepare data for 
+  :ref:`FFT <algm-FFT>` by applying padding and/or applying an apodization function. 
+
+Padding is when the input data is extended by adding extra measurments of zero at regular intervals. For real data this is only done after the end of the input data. However, for complex data the padding should be shared between the start and end of the input data.
+
+`Apodization functions <http://mathworld.wolfram.com/ApodizationFunction.html>`_ can be used to remove data with large errors. These are usualy
+found at large time scales. These take a decay constant 
+(:math:`\tau` ) that determines the rate at which the data goes to zero. 
+The time the function is evaluated at is denoted by :math:`t`.
+The current implementation includes the following functions:
+
+- None.
+- Bartlett :math:`1-\frac{\abs({t}}{\tau}`.
+- Lorentz :math:`\exp\left(-\frac{t}{\tau}\right)`.
+- Connes :math:`\left(1-\frac{t^2}{\tau^2}\right)^2`.
+- Cosine :math:`\cos\left(\frac{\pi t}{2 tau}\right)`.
+- Gaussian :math:`\exp\left(-\frac{t^2}{2\tau^2}\right)`.
+- Welch :math:`1-\frac{t^2}{\tau^2}`.
+ 
```

Comment:
I just checked the wolfram link. The decay constant must only be specified for Lorentz and Gaussian. For the other functions it is equal to the x-range.